src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -2328,6 +2328,12 @@ public override BoundNode VisitUnboundLambda(UnboundLambda node)
             return VisitLambda(node.BindForErrorRecovery());
         }
 
+        public override BoundNode VisitUnboundObjectCreationExpression(UnboundObjectCreationExpression node)
+        {
+            // The presence of this node suggests an error was detected in an earlier phase.
+            return null;
+        }
```

Comment:
You should visit the (bound) argument expressions here.

Reply:
and the initializer, I suppose. So both need to be bound earlier.